components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -871,7 +872,44 @@ def wrapObjectTemplate(templateBody, nullValue, isDefinitelyObject, type,
         return handleOptional(templateBody, declType, handleDefaultNull("None"))
 
     if type.isSpiderMonkeyInterface():
-        raise TypeError("Can't handle SpiderMonkey interface arguments yet")
```

Comment:
This is really for typed arrays, whereas `isSpiderMonkeyInterface()` also returns true for a `ReadableStream` type - should I explicitly handle only typed arrays here for now?